readme.md

```diff
@@ -268,6 +268,15 @@ zinit ice compile'(pure|async).zsh' pick'async.zsh' src'pure.zsh'
 zinit light sindresorhus/pure
 ```
 
+### [zi](https://wiki.zshell.dev)
```

Comment:
Would you mind removing `zinit` above as well? Zi seems to be the continuation of `zinit` (which is no more).

Reply:
I think that we can keep zinit and zi together, even if zi and zinit are forked from zdharma/zinit.

Reply:
Ok, if it's still in use then that's fine. 👍🏻